pa11y/webapp/write-report.js

```diff
@@ -1,15 +1,26 @@
+const chalk = require('chalk');
 const events = require('events');
 const fs = require('fs');
 const pa11y = require('pa11y');
 const { promisify } = require('util');
+const yargs = require('yargs');
 
 const writeFile = promisify(fs.writeFile);
 
 events.EventEmitter.defaultMaxListeners = 25;
 
+const { isPullRequestRun } = yargs(process.argv.slice(2))
```

Comment:
Perhaps we can push the notion of `isPullRequestRun` out of the script and hand the `baseUrl` directly?

Reply:
I'm not 100% on what you mean. How do we do that? 
This runs on Prod as well to update the dashboard, so we'd need some kind of way of knowing where this is being run to provide the correct url?

Reply:
One of the things `isPullRequestRun` is doing (set from a CLI arg) is setting the `baseUrl`. Instead of passing that flag, would it be simper to pass the value of `baseUrl` directly?